bananas_api/new_upload/readers/heightmap.py

```diff
@@ -50,11 +54,6 @@ def read(self, fp):
         except Exception:
             raise ValidationException("File is not a valid image.")
 
-        # Limit the size of heightmaps, as otherwise it could allocate a lot of
-        # memory for clients loading the map.
-        if im.width * im.height > 16384 * 16384:
-            raise ValidationException("Image is too large.")
```

Comment:
If I read this right, when you now upload an image that is too big, you get `File is not a valid image.`. This seems to be a bit misleading for an error. Is there any way we can give a better reply back to the user you think?

Reply:
Good question. I think we can use the exception type (aptly named `Image.DecompressionBombError`) to produce that user-friendly message.